src/ui/pages/GetStarted/ImportWallet/MnemonicImportView/AddressImportFlow/AddressImportFlow.tsx

```diff
@@ -131,6 +433,19 @@ function AddressImportList({
         </VStack>
         <PageBottom />
       </PageStickyFooter>
+      <SelectMoreWalletsDialog
+        initialValues={values}
+        wallets={wallets}
+        activeWallets={activeWallets}
+        existingAddressesSet={existingAddressesSet}
+        dialogRef={moreWalletsDialogRef}
+        onSubmit={(values) => {
+          const selectedWallets = wallets
+            .flatMap((c) => c.wallets)
+            .filter((wallet) => values.has(wallet.address));
+          onSubmit(selectedWallets);
```

Comment:
Tbh, I expected this dialog not to trigger global submit but just update values in the main screen. But this is my personal UX preferences

Reply:
I tried it like this and I liked that the views explicitly hold different sets of values